Apps/WebClient/src/ClientApp/app/components/timeline/medication.vue

```diff
@@ -164,14 +164,32 @@ $radius: 15px;
         <b-row>
           <b-col>
             <div class="d-flex flex-row-reverse">
-              <span class="py-2 px-0" v-if="this.comments.length > 0">{{
-                this.comments.length > 1
-                  ? this.comments.length + " comments"
-                  : "1 comment"
-              }}</span>
+              <b-btn
```

Comment:
Not to be done now: Further improvements could be made to move the entire comment list, including button, to a new component that takes the parent id and loads the comments and all is logic separately.

Reply:
Keeping in mind for further implementations!